src/Mayaqua/Encrypt.c

```diff
@@ -4138,6 +4138,22 @@ void FreeOpenSSLThreadState()
 	ERR_remove_state(0);
 }
 
+void FreeSsl()
+{
+	/* thread-safe cleanup */ 
+	ERR_remove_state(0);
+	ENGINE_cleanup(); 
+	CONF_modules_unload(1); 
+
+	sk_free(SSL_COMP_get_compression_methods());
+
+	/* global application exit cleanup (after all SSL activity is shutdown) */ 
+	ERR_free_strings(); 
+	EVP_cleanup(); 
+	CRYPTO_cleanup_all_ex_data();
+

```

Comment:
Extra empty line.

Unfortunately it can't be removed, as the source branch doesn't exist anymore.